### PR TITLE
Fix lisk_bridge to overwrite downloaded file - Closes #89

### LINKED ISSUE #89: Issues with lisk_bridge.sh
- usage of `f` flag is not documented
- `installLisk.sh` will not be overwritten if it exists (wget create `installLisk.sh.1` by default) and thus the old script will be used

### PR DESCRIPTION
**Pull request by nazarhussain**
Closes: #89

### CODE REVIEW

File: downloaded/lisk_bridge.sh [hunk 1]
```diff
@@ -45,9 +45,11 @@ fi
 if [[ $SHOW_USAGE ]]; then
 	echo "Usage: $0 <-h <BLOCKHEIGHT>> [-s <DIRECTORY>] [-n <NETWORK>] "
 	echo '-h <BLOCKHEIGHT> -- specify blockheight at which bridging will be initiated'
+	echo '-f <TARBALL> -- Specify local tarball for destination release'
```
Comment (fchavant) [inline downloaded/lisk_bridge.sh]: I don't think "destination release" is clear. Maybe "specify path to local tarball containing the target release"?

File: downloaded/lisk_bridge.sh [hunk 2]
```diff
@@ -45,9 +45,11 @@ fi
...
 	echo '-s <DIRECTORY>   -- Lisk home directory'
 	echo '-n <NETWORK>     -- choose main or test'
 	echo -e '\nExample: bash lisk_bridge.sh -h 50000000 -n test -s /home/lisk/lisk-test'
+	echo -e '\nUse LISK_MASTER_PASSWORD env variable if you want to do secrets migration in non-interactive mode'
```
Comment (fchavant) [inline downloaded/lisk_bridge.sh]: I would suggest "Set LISK_MASTER_PASSWORD environment variable [...]"

File: downloaded/lisk_bridge.sh [hunk 3]
```diff
@@ -75,6 +77,7 @@ if [[ "$BLOCK_HEIGHT" -gt "$TARGET_HEIGHT" ]]; then
 fi
 
 bash "$LISK_HOME/lisk.sh" stop
+rm installLisk.sh
```
Comment (fchavant) [inline downloaded/lisk_bridge.sh]: Should be `rm -f`; even better: `wget --no-clobber`

**Comment by nazarhussain** [comment]
@fchavant As this PR was closed, I created another PR to fix your feedback.

**Pull request merged**